examples/base/provisioned-plugin-auto-cd/push.yaml

```diff
@@ -45,13 +29,19 @@ jobs:
       # When pushing to "main", publish and deploy to Grafana Cloud (CD).
       # For PRs, skip publishing and deploying (run CI only).
       environment: ${{ (github.event_name == 'push' && github.ref_name == 'main') && env.CD_ENVIRONMENT || 'none' }}
-      scopes: ${{ env.SCOPES }}
+      scopes: grafana_cloud # TODO: Choose the appropriate scope for your plugin.
 
       # This will trigger the Argo Workflow to update your plugin version in Grafana Cloud via deployment_tools
       grafana-cloud-deployment-type: provisioned
-      argo-workflow-slack-channel: ${{ env.ARGO_WORKFLOW_SLACK_CHANNEL }}
+      argo-workflow-slack-channel: "#grafana-plugins-platform-ci" # TODO: Change with your own Slack channel
+      # Grafana Cloud environment to deploy the provisioned plugin to.
+      # - 'dev': dev
+      # - 'ops': ops
+      # - 'prod': dev + ops + prod
+      # Can also be a comma-separated list of environments, e.g. 'dev,ops'.
+      # Change to 'dev,ops' to deploy each push to main to both dev and ops environments.
       # Auto-merge all PRs in deployment_tools
-      auto-merge-environments: ${{ env.CD_ENVIRONMENT }}
+      auto-merge-environments: dev
```

Comment:
add the inline comment about choosing the appropriate (I know there's a big comment on top but devs will be devs)